main.go

```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	"flag"
+	_ "net/http/pprof"
```

Comment:
Do we need this, or was this temporary for debugging?

Reply:
I left it in deliberately. I don't think it should have any impact because this is not a performance critical application.